draft-ietf-tls-esni.md

```diff
@@ -777,9 +794,15 @@ ClientHelloOuter as follows:
 - `enc` is replaced with the empty string.
 - `payload` is replaced with the value computed above.
 
-If the client offered ECH in the first ClientHello, then it MUST offer ECH in
-the second. Likewise, if the client did not offer ECH in the first ClientHello,
-then it MUST NOT not offer ECH in the second.
```

Comment:
> if the client did not offer ECH in the first ClientHello, then it MUST NOT not offer ECH in the second.

Is this no longer true? If not, why not? If it is still true, did this get moved somewhere else?

Reply:
> If the client offered ECH in the first ClientHello, then it MUST offer ECH in
the second. Likewise, if the client did not offer ECH in the first ClientHello,
then it MUST NOT not offer ECH in the second.

Hmm, I think the first part of this paragraph is implied by the new rules. However, the second is not. I think I'll just revert this change. Good catch!